mlir/lib/Target/SPIRV/Deserialization/Deserializer.h

```diff
@@ -605,6 +614,9 @@ class Deserializer {
   /// A list of all structs which have unresolved member types.
   SmallVector<DeferredStructTypeInfo, 0> deferredStructTypesInfos;
 
+  /// A list of argument attributes of function.
```

Comment:
This should be a local variable in `processFunction` given this is only needed there and different functions have different argument attributes?